src/NavigationContainer.tsx

```diff
@@ -13,6 +13,7 @@ const MISSING_CONTEXT_ERROR =
 
 export const NavigationStateContext = React.createContext<{
   state?: NavigationState | InitialState;
+  getKey: () => string | null;
```

Comment:
Instead of `getKey`, it maybe better to just pass `key` as a property

Reply:
Maybe use `undefined` instead of `null`? xd

Reply:
I'd rather stay with `getKey` for a while. Idk why I cannot change it.